docs/tutorials/basic/symbol.md

```diff
@@ -160,7 +160,7 @@ composed.list_arguments()
 ```
 
 In this example, *net2* is used as a function to apply to an existing symbol *net1*,
-and the resulting *composed_net* will have all the attributes of *net1* and *net2*.
+and the resulting *composed* will have all the attributes of *net1* and *net2*.
```

Comment:
_composed_ is just in line with the actual code sample, and I felt the sample should take priority. But we can change example for sure. I like _composed_net_ the most. Sounds ok, or do you still prefer _composed_symbol_?